src/js/components/molecules/tabs/tabs.ts

```diff
@@ -61,7 +59,7 @@ export class Tabs extends Vue implements TabsProps {
         const tabs = this.$refs.tab;
         this.correction = tabList.scrollLeft;
 
-        if (!tabs) { return; }
+        if (!tabs) return;
```

Comment:
avoid these single line if statements. @see https://www.crockford.com/code.html#if%20statement

```suggestion
        if (!tabs) {
            return;
        }
```